varia/style.css

```diff
@@ -2436,35 +2440,35 @@ table th,
 }
 
 .has-primary-color[class] {
-	color: blue !important;
```

Comment:
Will the removal of these `!important` rules automatically flow down to the child themes when we recompile their CSS? It doesn't look like these rules are defined in the child theme CSS in the first place.